handlers/metrics.go

```diff
@@ -166,6 +166,69 @@ func getAggregateMetrics(w http.ResponseWriter, r *http.Request, promSupplier pr
 	RespondWithJSON(w, http.StatusOK, metrics)
 }
 
+// ControlPlaneMetrics is the API handler to fetch metrics to be displayed, related to a single control plane revision
+func ControlPlaneMetrics(promSupplier promClientSupplier, discovery remoteClusterIdentifier) http.HandlerFunc {
```

Comment:
I think you can now get rid of `discovery` from here and from the rest of the <...Metrics> functions as it was only for checking for the remote cluster and we no longer need to do that. ` remoteClusterIdentifier` can go too it looks like?

Reply:
You are correct, sir!  Weird, I'm surprised the IDE didn't flag that unused parameter.  Changes pushed.